.gitignore

```diff
@@ -27,6 +27,9 @@ annotation-file-utilities/javadoc
 annotation-file-utilities/tests/abbreviated/*.diff
 annotation-file-utilities/tests/abbreviated/*.log
 annotation-file-utilities/tests/abbreviated/*.output
+annotation-file-utilities/tests/enum-filed-imports/*.diff
```

Comment:
Yes, should be `field`.  This part `**/*.diff` should be ignored as they are the output of AFU testing.